src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -63,14 +63,48 @@ public static function load()
 
             $info = curl_getinfo($ch);
             $sanitizedUrl = Urls::sanitize($info['url']);
+            unset($info['url']);
+
             if ($globalConfig->isHttpClientSplitByDomain()) {
                 $span->setTag(Tag::SERVICE_NAME, Urls::hostnameForTag($sanitizedUrl));
             } else {
                 $span->setTag(Tag::SERVICE_NAME, 'curl');
             }
             $span->setTag(Tag::RESOURCE_NAME, $sanitizedUrl);
+
+
+            // Special case the Datadog Standard Attributes
+            //  https://docs.datadoghq.com/logs/processing/attributes_naming_convention/
+
             $span->setTag(Tag::HTTP_URL, $sanitizedUrl);
-            $span->setTag(Tag::HTTP_STATUS_CODE, $info['http_code']);
+
+            tagFromCurlInfo($span, $info, Tag::HTTP_STATUS_CODE, 'http_code');
+
+            // Datadog sets durations in nanoseconds - convert from seconds
+            $span->setTag('duration', $info['total_time'] * 1000000000);
+            unset($info['duration']);
+
+            tagFromCurlInfo($span, $info, 'network.client.ip', 'local_ip');
+            tagFromCurlInfo($span, $info, 'network.client.port', 'local_port');
+
+            tagFromCurlInfo($span, $info, 'network.destination.ip', 'primary_ip');
+            tagFromCurlInfo($span, $info, 'network.destination.port', 'primary_port');
+
+            tagFromCurlInfo($span, $info, 'network.bytes_read', 'size_download');
+            tagFromCurlInfo($span, $info, 'network.bytes_written', 'size_upload');
+
+
+            // Add the rest to a 'curl.' object
+            foreach ($info as $key => $val) {
+                // Datadog doesn't support arrays in tags 
+                if(!is_array($val)) {
+                  // Datadog sets durations in nanoseconds - convert from seconds
+                  if(substr_compare($key, '_time', -5) === 0) {
+                     $val = $val * 1000000000;
+                  }
+                  $span->setTag('curl.' . $key, $val);
+                }
```

Comment:
The linter is complaining about this bit. It wants a space after `if` and before `(`, and it wants four space indents (looks like you indented with 3 spaces).

Reply:
Lints clean now